### ISSUE TITLE
In webconsole I see message about that it can't load bar-card.js

### ISSUE THREAD

**Issue by fribse** [custom-cards/bar-card#86]
HA: 0.110.4
HACS: 1.0.3
Bar-card: 3.1.6

I have installed bar-card through HACS. I've had it installed for a while, but just noticed this message in the webconsole:
![image](https://user-images.githubusercontent.com/13660621/83409849-8d4b1d00-a415-11ea-882c-771e166f2b85.png)
I've tried uninstalling bar-card and searching for remnants, and removed a few, and then reinstall the bar card, but no change, it still shows up?
I have no idea if it's a bar-card issue or a lovelace issue, or maybe even a hacs issue?

**Comment by tomlut** [comment]
Seeing this as well.

**Comment by tomlut** [comment]
Also in the Ha log:
```
Tried to serve up '/config/www/community/bar-card/index.m.js.map' but it does not exist
```

**Comment by Gluwc** [comment]
Should be fixed in [3.1.7](https://github.com/custom-cards/bar-card/releases/tag/3.1.7).
